src/features/profile/Profile.tsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
-import {View, Text, Image, Switch} from 'react-native';
+import {View, Text, Image} from 'react-native';
 import {MaterialTopTabNavigationProp} from '@react-navigation/material-top-tabs';
 import {connect} from 'react-redux';
 import {ThunkDispatch} from 'redux-thunk';
 import {Action} from 'redux';
 import {useTheme} from '@react-navigation/native';
+import {RadioButton} from 'react-native-paper';
```

Comment:
We can implement RadioButton by ourselves or check library only for this component.